ogusa/tests/test_basic.py

```diff
@@ -81,7 +81,7 @@ def test_constant_demographics_TPI(dask_client):
                        ss_outputs['bssmat_splus1']))
 
 
-@pytest.mark.full_run
+@pytest.mark.full_local
```

Comment:
@jdebacker . I think you meant to use the `@pytest.mark.local` mark here. There is no defined `full_local` mark.

Reply:
@jdebacker . This is fixed. Marking resolved.